test/suites/dev-tanssi/issuance-rewards/test_issuance.ts

```diff
@@ -0,0 +1,35 @@
+import "@tanssi/api-augment";
+import { describeSuite, expect, beforeAll } from "@moonwall/cli";
+import { ApiPromise } from "@polkadot/api";
+import { fetchIssuance } from "util/block";
+
+describeSuite({
+    id: "IR0402",
+    title: "Issuance reward test suite",
+    foundationMethods: "dev",
+    testCases: ({ it, context }) => {
+        let polkadotJs: ApiPromise;
+        beforeAll(async () => {
+            polkadotJs = context.polkadotJs();
+        });
+        it({
+            id: "E01",
+            title: "Issuance is the correct percentage",
+            test: async function () {
+                const supplyBefore = (await polkadotJs.query.balances.totalIssuance()).toBigInt();
+                await context.createBlock();
+
+                const events = await polkadotJs.query.system.events();
+
+                const issuance = await fetchIssuance(events).amount.toBigInt();
+
+                const supplyAfter = (await polkadotJs.query.balances.totalIssuance()).toBigInt();
+
+                // in dev mode is 1%
```

Comment:
Is it enough to only test the issuance of dev mode?

Reply:
this is a good point. I am not entirely sure why we have different values for dev and prod, I will try to unify

Reply:
we will leave it as is for now, I dont want to break any tests. But food for thought for the future